numba_dppy/rename_numpy_functions_pass.py

```diff
@@ -133,6 +163,7 @@ def __init__(self):
         import numba_dppy.dpnp_glue.dpnp_transcendentalsimpl
         import numba_dppy.dpnp_glue.dpnp_statisticsimpl
         import numba_dppy.dpnp_glue.dpnp_sort_search_countimpl
+        import numba_dppy.dpnp_glue.dpnp_randomimpl
```

Comment:
Is it importing for registering overloads to numba?

Reply:
These are registering the overloads to numba.